dotnet3.5/S3/ListObjectsExample/ListObjects/ListObjects.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="AWSSDK.Core" Version="3.5.1.32" />
```

Comment:
I'm wondering if its necessary to add .csproj and .sln files to the repo, especially when these package versions are so specific (down to the w.x.y.z level) that in a few days from now, they'll be out of date.

Reply:
I agree, especially if you only use .NET Core, both are supported on Linux and Mac.

Reply:
I don't see any reason to not include them. It allows less experienced developers to download the project or the solution and run it easily. The examples as I create them will use .NET 5. When we get to the point of having the developers review the code, maybe they will have an opinion on this.